com.outerwildsmods.owmods_gui.yml

```diff
@@ -0,0 +1,82 @@
+app-id: com.outerwildsmods.owmods_gui
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.mono6
+command: outer-wilds-mod-manager
+rename-icon: outer-wilds-mod-manager
+finish-args:
+    - --socket=fallback-x11
+    - --socket=wayland
+    - --share=ipc
+    - --share=network
+    - --filesystem=home
```

Comment:
Why does it need full home access, instead of just the path to Outer Wilds?

Reply:
In order to keep backwards compatibility with our old manager, the app needs to access `~/.local/share/OuterWildsModManager`. However I see that `home` is too broad and updated the manifest accordingly.